### DISCUSSION TITLE
Is there a transcode cache? and how to delete it?

### DISCUSSION THREAD

**Discussion by djotaku** [ampache/ampache#3565] [Q&A]
I was having the issue where transcoding flac cuts off the last bit of the file (usually the last 15 seconds or so). I turned off flac transcoding, but I'm still having the issue. Is there a cache somewhere? If so, how do I delete it?

Thanks!

**Comment by lachlan-00** [comment]
on the fly transcoding is piped so it doesn't cache and is transcoded each time. There are issues calculating the end of the stream and the output can get truncated.

I have certain albums that do this so i pre-cache the transcoded files so when you stream the complete file comes through.
https://github.com/ampache/ampache/wiki/Transcode-Caching

I also rewrote the playback class into play2 but fixing these stream issues caused issues for other people in different ways so i've separated it.

turn it on in the streaming options if you want to try that first.
![image](https://github.com/ampache/ampache/assets/1305249/d0155a85-d289-4a29-aab5-d3ebbfbff6c2)
